board.py

```diff
@@ -0,0 +1,268 @@
+            counter += 1
+        print(str(counter) + '. ' + 'Main Menu')
+        userchoice = input('Choose chirp to reply \n> ')
+        if userchoice == str(counter):
+            self.clearScreen()
+            self.menu()
+        else:
+            self.getListofChirps()
+            message = self.listofChirps[int(userchoice) - 1]
+            self.clearScreen()
+            self.replyMenu(message)
+
+    def getListofChirps(self):
+        """
+        Appends to a list all of a users private and public chirps.
+        Used to find the corresponding chirp in self.chirps
+        """
+        self.findPrivateChirps()
+        try:
+            for key, value in self.userPrivateChirps.items():
+                self.listofChirps.append(value)
+        except AttributeError:
+            pass
+        try:
+            for key, value in self.chirps['public'].items():
+                self.listofChirps.append(value)
+        except AttributeError:
+            pass
+        return self.listofChirps
+
+
+    def findPrivateChirps(self):
+        """Find current users private chirps"""
+        for key, value in self.chirps['private'].items():
+            if (self.currentUser.username == value.sender) or (self.currentUser.username == value.receiver):
+                self.userPrivateChirps[key] = value
+                return self.userPrivateChirps
+            else:
+                self.userPrivateChirps = dict()
+                return self.userPrivateChirps
+
+    def printConversation(self, message):
+        """Print's an individual chirp's reply's"""
+        print(message.sender + ': ' + message.message)
+        for reply in message.conversation:
+            for key in reply:
+                print(key + ': ' + reply[key])
+
+    def replyMenu(self, conversation):
+        """Prompts the user to either reply to message or go back to chirp menu"""
+        self.clearScreen()
+        self.printConversation(conversation)
+        userchoice = input('1. Reply \n2. Go Back \n> ')
+        if userchoice == '1':
+            self.clearScreen()
+            newReply = input('Enter Reply \n> ')
+            self.reply(newReply, conversation)
+            self.clearScreen()
+            self.replyMenu(conversation)
+        elif userchoice == '2':
+            self.clearScreen()
+            self.chirpMenu()
+
+    def reply(self, reply, conversation):
+        """Searches for chosen message in message file and appends a reply to its conversation list"""
+        for key, value in self.chirps['public'].items():
+            if value == conversation:
+                value.reply(self.currentUser.username, reply)
+                self.serializeMessages(self.boardfile)
+        for key, value in self.chirps['private'].items():
+            if value == conversation:
+                value.reply(self.currentUser.username, reply)
+                self.serializeMessages(self.boardfile)
+
+    def clearScreen(self):
```

Comment:
I like that you've pulled this out into its own method, makes both the intent clearer than the os call, and allows you to add other things should you want the clearScreen to do additional actions in the future.